app/views/idv/inherited_proofing/get_started.html.erb

```diff
@@ -8,12 +8,39 @@
   <p>
     <%= t('inherited_proofing.info.welcome_html', sp_name: decorated_session.sp_name || t('inherited_proofing.info.no_sp_name')) %>
   </p>
-  <h2><%= t('inherited_proofing.instructions.welcome') %></h2>
 
-  <%= render ProcessListComponent.new(heading_level: :h3, class: 'margin-y-3') do |c| %>
-    <%= c.item(heading: t('inherited_proofing.instructions.bullet1')) %>
-  <% end %>
+  <%= t('inherited_proofing.instructions.welcome') %>
+  <b>
+    <%= t('inherited_proofing.instructions.verify_requirements') %>
+  </b>
+  <br/>
+  <div>
+    <br/>
+    <%= t('inherited_proofing.instructions.verification_content') %>
+  </div>
 
+  <%= render(
+          'shared/troubleshooting_options',
+          heading_tag: :h3,
+          heading: t('inherited_proofing.troubleshooting.headings.missing_required_items'),
+          options: [
+            {
+              url: 'https://www.va.gov/resources/managing-your-vagov-profile/',
```

Comment:
I feel like the url should be hidden away in a decorator at some point. Perhaps in future clean up.